# file: C:\Users\Couch\AppData\Roaming\Python\Python313\site-packages\httpx\_exceptions.py
# hypothesis_version: 6.141.1

['CloseError', 'ConnectError', 'ConnectTimeout', 'CookieConflict', 'DecodingError', 'HTTPError', 'HTTPStatusError', 'InvalidURL', 'LocalProtocolError', 'NetworkError', 'PoolTimeout', 'ProtocolError', 'ProxyError', 'ReadError', 'ReadTimeout', 'RemoteProtocolError', 'RequestError', 'RequestNotRead', 'ResponseNotRead', 'StreamClosed', 'StreamConsumed', 'StreamError', 'TimeoutException', 'TooManyRedirects', 'TransportError', 'UnsupportedProtocol', 'WriteError', 'WriteTimeout']